### ISSUE TITLE
Update T1w scans with re-defaced images

### ISSUE THREAD

**Issue by renelabounek** [spine-generic/data-multi-subject#146]
Following the #115, I have shared with @jcohenadad 49 re-defaced T1w scans. I have done visual inspection and confirm that no subject ID switch happenned and that re-defaced scans contains the whole brain volume without any cut-offs.

Six scans can not be re-defaced, because raw data are unavailable:
- sub-milan05
- sub-milan06
- sub-milan07
- sub-beijingVerio01
- sub-beijingVerio02
- sub-beijingVerio04

We should decide if bain cut offs are minimal in these 6 scans or if these IDs should be added into the brain_t1 category in the [exclude.yml](https://github.com/spine-generic/data-multi-subject/blob/master/exclude.yml) file.

I am still awaiting if another 4 scans of one center will be re-defaced or not.

**Comment by valosekj** [comment]
Thank you @renelabounek!
I successfully downloaded the re-defaced images for 49 subjects and saved them to `~/duke/temp/janvalosek/data-multi-subject-redefaced`.

<s>@mguaypaq, could you please push the data to git-annex? Thanks!</s>

**Comment by renelabounek** [comment]
In the most of the images in bids format, I have executed fslreorient2std. But not in all the cases. Thus, it may be worth to double check if orientation of all updated scans is the same as manually drawn masks, etc. in the database.

**Comment by valosekj** [comment]
> In the most of the images in bids format, I have executed fslreorient2std.

Thanks for this comment! This is important information!

I checked the orientation of both the original images (located under [master branch](https://github.com/spine-generic/data-multi-subject)) and the re-defaced ones (`~/duke/temp/janvalosek/data-multi-subject-redefaced`):

```console
# orig
data-multi-subject-redefaced$ for file in sub-*/anat/*nii.gz;do echo ${file}; sct_image -i ../data-multi-subject/${file} -header | grep -E qform_[xyz];echo "";done >> orientations_orig.txt

# re-defaced
data-multi-subject-redefaced$ for file in sub-*/anat/*nii.gz;do echo ${file}; sct_image -i ${file} -header | grep -E qform_[xyz];echo "";done >> orientations_redefaced.txt
```

[orientations_orig.txt](https://github.com/spine-generic/data-multi-subject/files/12306384/orientations_orig.txt)
[orientations_redefaced.txt](https://github.com/spine-generic/data-multi-subject/files/12306385/orientations_redefaced.txt)

**Indeed, in 18 subjects, orientation has been changed from LPI to RPI**; see [here](https://www.diffchecker.com/YPzdOpSG/).

> Thus, it may be worth to double check if orientation of all updated scans is the same as manually drawn masks, etc. in the database.

Good point! The thing is that the label files under [`derivatives/labels`](https://github.com/spine-generic/data-multi-subject/tree/master/derivatives/labels) already have a different orientation (and sometimes even resolution!!!) relative to the original images. This happened because the images were reoriented to RPI and resampled to 1mm iso, see code [here](https://github.com/spine-generic/spine-generic/blob/c8c8408715f101414ca08c6bcf29a5d5b87df90a/process_data.sh#L149-L151). See also issue https://github.com/spine-generic/data-multi-subject/issues/144.

---

**A possible solution:** reorient the re-defaced images back to LPI using `sct_image -setorient LPI` to match the original images under [master branch](https://github.com/spine-generic/data-multi-subject).

**Comment by valosekj** [comment]
> **A possible solution:** reorient the re-defaced images back to LPI using `sct_image -setorient LPI` to match the original images under [master branch](https://github.com/spine-generic/data-multi-subject).

We agreed on this solution during the SCT dev meeting on 2023-08-10 --> I will do it.

**Comment by valosekj** [comment]
<details><summary>I fixed the orientation using the following script:</summary>

```bash
#!/bin/bash

# Fix orientation of the re-defaced images to match the original images

for file in sub-*/anat/*nii.gz;do
	
	redefaced_image=${file}
	original_image="../data-multi-subject/${file}"

	echo "Redefaced image: ${redefaced_image}"
	echo "Original image: ${original_image}"

	# Fetch orientations, e.g., LPI or RPI
	redefaced_orientation=$(sct_image -i ${redefaced_image} -header | grep -E qform_[xyz] | awk '{printf "%s", substr($2, 1, 1)}')
	original_orientation=$(sct_image -i ${original_image} -header | grep -E qform_[xyz] | awk '{printf "%s", substr($2, 1, 1)}')

	echo "Redefaced orientation: ${redefaced_orientation}"
	echo "Original orientation: ${original_orientation}"

	if [[ ${redefaced_orientation} != ${original_orientation} ]]; then
		echo "Orientations are different --> fixing it"
		sct_image -i ${redefaced_image} -setorient ${original_orientation} -o ${redefaced_image}
		fixed_orientation=$(sct_image -i ${redefaced_image} -header | grep -E qform_[xyz] | awk '{printf "%s", substr($2, 1, 1)}')
		echo "Fixed orientation: ${fixed_orientation}"
	fi

done
```

</details>

Now, the orientation between the re-defaced and original images (located under [master branch](https://github.com/spine-generic/data-multi-subject)) is identical:

[orientations_orig.txt](https://github.com/spine-generic/data-multi-subject/files/12321344/orientations_orig.txt)
[orientations_redefaced.txt](https://github.com/spine-generic/data-multi-subject/files/12321345/orientations_redefaced.txt)

See diff [here](https://www.diffchecker.com/ofCl86ZT/).

I saved the data to `~/duke/temp/janvalosek/data-multi-subject-redefaced`. @mguaypaq, could you please push the data to git-annex? Thanks!
